src/App.js

```diff
@@ -116,4 +130,4 @@ class AlbumCover extends Component {
   }
 }
 
-export default App;
+export default App;
```

Comment:
Il manque une ligne à la fin du fichier :) 
Normalement si tu as configuré ton IDE pour il le fera automatiquement on save
[Des détails sur le why](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)